### ISSUE TITLE
M249 is a joke weapon

### ISSUE THREAD

**Issue by visplaneoverflow** [eezstreet/SWATEliteForce#657]
In its present state the M249 is not very usable. Many in the community will refer to it as a joke weapon or a meme weapon, largely because after the first shot, the recoil on the weapon makes your rounds go all over the place, even in very short bursts. The player would expect the SAW to have less recoil than light weapons in the same cartridge. It appears also that the intention was for the SAW to have light recoil but a long aiming convergence time.

This seems to be due to the fact that the AutoRecoilPerShot value for the M249 is set very high, approximately 2 and a half times the value of the M16. For context, on the M16, AutoRecoilPerShot=38.66. On the SAW, AutoRecoilPerShot=94.77.

Expected behaviour: M249 should experience less recoil than other 5.56 weapons by virtue of being a heavier weapon

Actual behaviour : M249 experiences more recoil than other 5.56 weapons.

Suggested solution : Reduce the AutoRecoilPerShot value below the other 5.56 weapons.

I don't think this change will affect game balance negatively because the M249's additional firepower is already offset by its extreme weight and bulk, preventing the player from equipping extra equipment or heavy body armour.

**Comment by eezstreet** [comment]
@kevinfoley are you planning on merging your change in with your PR?

**Comment by kevinfoley** [comment]
@eezstreet Yeah, I'm working on the weapons in a branch but haven't created a PR because I'm not done yet.
